Source/WebKit/UIProcess/Launcher/playstation/ProcessLauncherPlayStation.cpp

```diff
@@ -97,8 +97,8 @@ void ProcessLauncher::launchProcess()
 
     // We've finished launching the process, message back to the main run loop.
     RefPtr<ProcessLauncher> protectedThis(this);
-    RunLoop::main().dispatch([=] {
-        protectedThis->didFinishLaunchingProcess(appLocalPid, IPC::Connection::Identifier { WTFMove(socketPair.server) });
+    RunLoop::main().dispatch([&] {
+        protectedThis->didFinishLaunchingProcess(appLocalPid, IPC::Connection::Identifier({ socketPair.server.value(), UnixFileDescriptor::Adopt }));
```

Comment:
This isn't right because now both UnixFileDescriptor objects hold ownership of the same fd; whichever one gets destroyed second will have a "dangling reference" to a closed fd, which it will also call `close()` on when destroyed. I think `WTFMove()` was correct. (You could also do `{ socketPair.server.release(), UnixFileDescriptor::Adopt }` to transfer the native fd from the old UnixFileDescriptor to the new one.)

Don has already fixed this in https://github.com/WebKit/WebKit/pull/38175 though.